### PR TITLE
test(ticdc): use mysql 2022 rpm GPG key

### PR DESCRIPTION
**Pull request by hi-rustin**
### What problem does this PR solve?

Issue Number: close https://github.com/pingcap/tiflow/issues/4433

### What is changed and how it works?

use mysql 2022 rpm GPG key.

### Check List 

Tests 

 - No code

Code changes

 None

Side effects

 None

Related changes

 - Need to cherry-pick to the release branch

### Release note 

```release-note
 None
```

### CODE REVIEW

**Review by 3AceShowHand — APPROVED** [review]

File: deployments/ticdc/docker/integration-test.Dockerfile [hunk 1]
```diff
@@ -39,9 +39,11 @@ RUN yum install -y \
 RUN wget http://dl.fedoraproject.org/pub/epel/epel-release-latest-7.noarch.rpm
 RUN yum install -y epel-release-latest-7.noarch.rpm
 RUN yum --enablerepo=epel install -y s3cmd
-RUN wget -i -c http://dev.mysql.com/get/mysql57-community-release-el7-10.noarch.rpm
-RUN yum install -y mysql57-community-release-el7-10.noarch.rpm
-RUN yum install -y mysql-server
+# Install mysql client.
+RUN rpm -ivh https://repo.mysql.com//mysql57-community-release-el7-11.noarch.rpm
```
Comment (lance6716) [inline deployments/ticdc/docker/integration-test.Dockerfile:43]:
```suggestion
RUN rpm -ivh https://repo.mysql.com/mysql57-community-release-el7-11.noarch.rpm
```

File: scripts/canal/docker/Dockerfile [hunk 2]
```diff
@@ -8,8 +8,9 @@ RUN yum clean all
 RUN yum makecache
 
 # Install mysql client.
-RUN wget https://dev.mysql.com/get/mysql57-community-release-el6-9.noarch.rpm
-RUN rpm -ivh mysql57-community-release-el6-9.noarch.rpm
+RUN rpm -ivh https://repo.mysql.com//mysql57-community-release-el7-11.noarch.rpm
```
Comment (lance6716) [inline scripts/canal/docker/Dockerfile:11]:
```suggestion
RUN rpm -ivh https://repo.mysql.com/mysql57-community-release-el7-11.noarch.rpm
```

**Comment by hi-rustin** [comment]
@lance6716 
Thanks for your review!  💚 💙 💜 💛 ❤️

**Review by lance6716 — APPROVED** [review]

**Pull request merged**